src/Standards/Generic/Sniffs/CodeAnalysis/UselessOverridingMethodSniff.php

```diff
@@ -93,30 +101,29 @@ public function process(File $phpcsFile, $stackPtr)
         $next = $phpcsFile->findNext(Tokens::$emptyTokens, ($next + 1), null, true);
 
         // Skip for invalid code.
-        if ($next === false || $tokens[$next]['code'] !== T_DOUBLE_COLON) {
+        if ($tokens[$next]['code'] !== T_DOUBLE_COLON) {
             return;
         }
 
-        // Find next non empty token index, should be the function name.
+        // Find next non empty token index, should be the parent method name.
```

Comment:
```suggestion
        // Find next non empty token index, should be the method name.
```

Reply:
I'm not sure if I understand this change. This call to `findNext()` returns the token that corresponds to the parent method name instead of the method name, no?

Using the following code from the test as an example:

```
public function differentParentMethod($a, $b) {
    return parent::anotherMethod($a, $b);
}
```

`findNext()` returns the token for `anotherMethod` and not the token for `differentParentMethod()`.

Reply:
> I'm not sure if I understand this change. This call to `findNext()` returns the token that corresponds to the parent method name instead of the method name, no?

Well, yes and no. It's the method name _being called_, but still a method name all the same and whether the method exists on the parent is something this sniff will not have any awareness of.

Would "should be the name of the method being called." work for you and make things clearer ?

Reply:
Oh, when writing the code comment and in my previous comment here, I completely missed that the method could be calling any method from the parent class. Now, your original suggestion sounds good to me, but the last one sounds a bit better, so I went with it.